src/Bookings.js

```diff
@@ -1,34 +1,83 @@
-import React from "react";
+import React, {useState,useEffect} from "react";
 import Footer from "./Footer.js";
 import Search from "./Search.js";
 import SearchResults from "./SearchResults.js";
-import FakeBookings from "./data/fakeBookings.json";
+// import FakeBookings from "./data/fakeBookings.json";
 import TouristInfoCards from "./TouristInfoCards.js";
 
 
 
 
+const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [dataLoading, setDataLoading] = useState(true);
+  const [error, setError] = useState(false);
 
+  // useEffect(() => {
+  //   fetch("https://cyf-react.glitch.me/delayed").then(res =>
+    // fetch(`https://cyf-react.glitch.me`).then(res =>
+    // fetch(`https://cyf-react.glitch.me`)
+      // fetch("https://cyf-react.glitch.me/delayed")
+      // fetch("https://cyf-react.glitch.me/error")
+  //     res.json().then(data => {
+  //       console.log(data);
+  //       setBookings(data);
+  //     })
+  //   );
+  // }, []);
+ 
 
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me/delayed")
 
-const Bookings = () => {
+    //  fetch("https://cyf-react.glitch.me")
+   
+    // fetch("https://cyf-react.glitch.me/error")
+    
+      .then(res => {
+        if (res.ok) {
+          return res.json();
+        }
+        throw Error("Error fetching data");
+      })
+      .then(data => {
+        setBookings(data);
+        setDataLoading(false);
+      })
+      .catch(err => {
+        console.log(err.message);
+        setError(true);
```

Comment:
You'll also want to set `setDataLoading(false` when there's an error too, as it's completed the request